server/api/cluster_test.go

```diff
@@ -56,12 +56,8 @@ func (s *testClusterSuite) TestCluster(c *C) {
 
 	c2 := &metapb.Cluster{}
 	r := config.ReplicationConfig{
-		EnablePlacementRules: true,
```

Comment:
Why remove it?

Reply:
This is the "only one default rule when enable placement rules", so `c.Assert(s.svr.SetReplicationConfig(r), NotNil)` is no longer correct here.

And the similar test is already done in `config_test`, so I think just remove the code to simply fix the test is ok.